### PR TITLE
Readme scraper

### PR DESCRIPTION
**Pull request by l0k0ms**
Overview
----

To reduce duplicated content, all integrations related content should be in their specific integrations-core/dogweb repo readme.

This is directly inspired from the process `{{<get-metrics-from-git>}}` 

** For now this only works for integration core repo, but this will be extended to dogweb repo at some point **

Integration's Readme have all the following sections:

```
## Overview
### Setup
#### Installation 
#### Configuration
#### Validation

## Data Collected
### Service checks
### Events
### Metrics

## Troubleshooting
### faq_1 ?
### faq_2 ?

## Further Reading 
### KB 
### Blog Article 
```

The idea here is to have a macro like:
`{{< get-SECTION-from-git >}}` that inline the corresponding content from the readme file in the doc page.

### CODE REVIEW

**Comment by l0k0ms** [comment]
Preview: https://d2cx6t4ssmwdrv.cloudfront.net/gus/readme-scraper/integrations/activemq/

File: local/bin/py/update_pre_build.py [hunk 1]
```diff
@@ -121,6 +156,57 @@ def parse_args(args=None):
     parser.add_option("-s", "--source", help="location of src files", default=curdir)
     return parser.parse_args(args)
 
+
+def readme_get_section(from_path,key_name):
```
Comment (l0k0ms) [inline local/bin/py/update_pre_build.py]: nit: readme_get_sections

File: content/integrations/activemq.md [hunk 2]
```diff
@@ -9,103 +9,23 @@ description: "{{< get-desc-from-git >}}"
 ---
 
 ## Overview
-
-Get metrics from ActiveMQ in real time to
-
-* Visualize your web ActiveMQ server performance
-* Correlate the performance of ActiveMQ with the rest of your applications
+{{< get-overview-from-git >}}
```
Comment (mstbbs) [inline content/integrations/activemq.md]: @l0k0ms - this is all looking good. We should decide on a different pattern for front-matter placeholders. The pattern you are using now {{< get-desc-from-git >}} is currently being used by Hugo to designate shortcodes, and so may be confusing to other developers. In the python script you are using this pattern for substitution. Maybe something like << desc-from-git >> ?

Comment (l0k0ms) [reply content/integrations/activemq.md]: 👍

File: content/integrations/activemq.md [hunk 3]
```diff
@@ -9,103 +9,23 @@ description: "{{< get-desc-from-git >}}"
...
 
 {{< get-metrics-from-git "activemq_xml" >}}
 
+### Events
+
+The Activemq check does not include any event at this time.
+
+### Service Checks
+
+The Activemq check does not include any service check at this time.
```
Comment (kmshultz) [inline content/integrations/activemq.md]: > The Activemq check does not include any service check at this time.

"The ActiveMQ check does not submit any service checks."

Anyway, don't we want to put this in integrations-core, not here?

File: content/integrations/activemq.md [hunk 4]
```diff
@@ -9,103 +9,23 @@ description: "{{< get-desc-from-git >}}"
...
+//get-setup-from-git//
 
 ## Data Collected
 ### Metrics
 
 {{< get-metrics-from-git "activemq_xml" >}}
 
+### Events
+
+The Activemq check does not include any event at this time.
```
Comment (kmshultz) [inline content/integrations/activemq.md]: > The Activemq check does not include any event at this time.

The ActiveMQ check does not emit any events.

File: local/bin/py/update_pre_build.py [hunk 5]
```diff
@@ -57,6 +73,24 @@ def download_github_files(token, org, repo, branch, to_path, is_dogweb=False):
         print('There was an error ({}) listing {}/{} contents..'.format(response.status_code, repo, branch))
         exit(1)
 
+    """
+    Donwloading readme.md for integrations core repo only
```
Comment (kmshultz) [inline local/bin/py/update_pre_build.py]: nit: s/Donwloading/Downloading

File: content/integrations/activemq.md (hunk 3 above)
Comment (l0k0ms) [reply content/integrations/activemq.md]: yes you are right :)

**Pull request merged**